test/utilities/commandPrompt.ts

```diff
@@ -184,3 +183,8 @@ export async function clickFilePathOkButton(): Promise<void> {
   }
   await pause(Duration.seconds(2));
 }
+
+export async function setDefaultOrg(targetOrg: string): Promise<void> {
+  const inputBox = await executeQuickPick('SFDX: Set a Default Org');
+  await findQuickPickItem(inputBox, targetOrg, false, true);
+}
```

Comment:
I don't think this helper function should belong in `commandPrompt.ts`.  Can we make a new utility file `authentication.ts` or at least move the function to `miscellaneous.ts`?